ui/src/components/footer/LinkGrid.js

```diff
@@ -72,6 +72,9 @@ export function LinkGrid() {
           <Link href="https://hongkong.sub.hmcc.net/privacy-policy-2/">
             <Text fontWeight="bold">Privacy Policy</Text>
           </Link>
+          <Link href="https://hongkong.sub.hmcc.net/urgent-announcements/hmcc-covid-19-safety-precautions/">
```

Comment:
<img width="1680" alt="Screenshot 2022-10-15 at 16 18 29" src="https://user-images.githubusercontent.com/33679483/195976827-28a6ce34-a6e3-4179-89c5-d88204342a7b.png">

can we also add the same thing here?

Reply:
okay good point